content/docs/reference-react-component.md

```diff
@@ -29,111 +28,111 @@ class Welcome extends React.Component {
 }
 ```
 
-The only method you *must* define in a `React.Component` subclass is called [`render()`](#render). All the other methods described on this page are optional.
+El único método que *debes* definir en un `React.Component` subclase es [`render()`](#render). Todos los métodos descritos en esta página son opcionales.
 
-**We strongly recommend against creating your own base component classes.** In React components, [code reuse is primarily achieved through composition rather than inheritance](/docs/composition-vs-inheritance.html).
+**Recomendamos encarecidamente no crear tus propias clases de componentes base.** En los componentes de React, [la re utilización de código se consigue principalmente a través de la composición en lugar de heredar](/docs/composition-vs-inheritance.html).
 
->Note:
->
->React doesn't force you to use the ES6 class syntax. If you prefer to avoid it, you may use the `create-react-class` module or a similar custom abstraction instead. Take a look at [Using React without ES6](/docs/react-without-es6.html) to learn more.
+> Nota:
+> 
+> React no obliga a usar la sintaxis de clases ES6. Si prefiere evitarlo, puede utilizar el módulo `create-react-class` o una abstracción personalizada similar en su lugar. Échale un vistazo a [Usando React sin ES6](/docs/react-without-es6.html) para aprender mas.
 
-### The Component Lifecycle
+### El Ciclo de Vida del Componente
 
-Each component has several "lifecycle methods" that you can override to run code at particular times in the process. **You can use [this lifecycle diagram](http://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/) as a cheat sheet.** In the list below, commonly used lifecycle methods are marked as **bold**. The rest of them exist for relatively rare use cases.
+Cada componente tiene varios "métodos de ciclo de vida" que puedes sobrescribir para ejecutar código en momentos particulares del proceso. **Puedes usar [este diagrama de ciclo de vida](http://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/) como una hoja de referencia.** En la lista de abajo, los métodos de ciclo de vida comúnmente usados están marcados en **negrita**. El resto de ellos existen para casos de uso relativamente raros.
 
-#### Mounting
+#### Montaje
 
-These methods are called in the following order when an instance of a component is being created and inserted into the DOM:
+Estos métodos se llaman cuando se crea una instancia de un componente y se inserta en el DOM:
 
-- [**`constructor()`**](#constructor)
-- [`static getDerivedStateFromProps()`](#static-getderivedstatefromprops)
-- [**`render()`**](#render)
-- [**`componentDidMount()`**](#componentdidmount)
+* [**`constructor()`**](#constructor)
```

Comment:
Why did you use `*` instead of `-`?

Reply:
This was in Crowdin translate, i think we should check if "original" md in crowdin is correct

Reply:
We should use `Crowdin` as a starting point and not the source of truth.

That said, I consider a good practice to reread Crowdin's content after pasting it, also, if you ran prettier and it changed `-` to `*`, that's totally OK.

Reply:
Crowdin is off of an old branch too, so it might have been that this file used to use `*` but was updated to use `-` at some point recently.

Reply:
@tesseralis could we update Crowdin translate so my only job is to "merge" both of them and translate the new changes?

Reply:
@semoal I don't think that Crowdin is the source of truth right now, so I don't think it will get updated frequently. That said, you should not rely on Crowdin to make any progress on this (or any other translation).

@tesseralis correct me if I'm wrong!

Reply:
@alejandronanez you’re right